### PR TITLE
Fix bug for detecting last visible CFI

### PR DESCRIPTION
**Pull request by TriangularCube**

### CODE REVIEW

File: src/navigator/views/content-view/r2-single-page-content-view.ts [hunk 1]
```diff
@@ -118,6 +118,7 @@ export class R2SinglePageContentView extends R2ContentView  {
 
     const cfi = this.cfiNavLogic.getFirstVisibleCfi(new Rect(left, top, right, bottom), backward);
 
+
```
Comment (denglili-ep) [inline src/navigator/views/content-view/r2-single-page-content-view.ts]: please remove extra space to avoid unnecessary change

Comment (TriangularCube) [reply src/navigator/views/content-view/r2-single-page-content-view.ts]: Got it
